crates/sui-framework/packages/sui-framework/sources/balance.move

```diff
@@ -6,6 +6,8 @@
 /// custom coins with `Supply` and `Balance`s.
 module sui::balance;
 
+use sui::accumulator;
+
```

Comment:
nit, maybe just fully qualify the call sites, or make the use limited to the functions they are used (since they are subject to some churn it might be best to localize them)